tests/dataclasses/validataclass_test.py

```diff
@@ -62,9 +59,9 @@ class UnitTestValidatorDataclass:
 
         # Check names and types of all fields
         assert list(fields.keys()) == ['foo', 'bar', 'baz']
-        assert fields['foo'].type is int
-        assert fields['bar'].type is int
-        assert fields['baz'].type is Optional[str]
+        assert fields['foo'].type == int
```

Comment:
Just out of interest: why did you change this? :)

Reply:
You mean changing the `is` to `==`?

One reason is that mypy was complaining about it (see the `# mypy: no-strict-equality` in the file header that I have removed now).

But also, `==` is sufficient anyway and it makes working with `Union` and other constructs easier. For example, `int | str` and `typing.Union[int, str]` are equal (in the sense of `==`), but technically they aren't the same thing (in the sense of `is`) because they're represented by different classes:

```py
>>> from typing import Union
>>> int | str == Union[int, str]
True
>>> int | str is Union[int, str]
False
>>> type(Union[int, str])
<class 'typing._UnionGenericAlias'>
>>> type(int | str)
<class 'types.UnionType'>
```

Reply:
Ui. Interesting :)